app/controllers/staffs/dashboard_controller.rb

```diff
@@ -10,11 +9,78 @@ def index
       @download = Quiz::LESSON_VALUES.map { |n| ["Lesson #{n}", n] }
     end
 
+    def import
+      @results = []
+    end
+
+    def import_reader
+      @logins = params[:logins]
+      @results = @logins.split
+                        .reject { |login| Reader.find_by_login login }
+                        .map { |login| import_staff login, Reader.new }
+      respond_to do |format|
+        format.html { redirect_to import_staffs_dashboard_index_path }
+        format.csv do
+          send_data User.create_csv(@results),
+                    filename: 'readers.csv'
+        end
+      end
+    end
+
+    def import_gsi
+      @logins = params[:logins]
+      @results = @logins.split
+                        .reject { |login| Gsi.find_by_login login }
+                        .map { |login| import_staff login, Gsi.new }
```

Comment:
Style/DotPosition: Place the . on the previous line, together with the method call receiver.